packages/app/components/drop/drop-free.tsx

```diff
@@ -49,54 +50,61 @@ const SECONDS_IN_A_DAY = 24 * 60 * 60;
 const SECONDS_IN_A_WEEK = 7 * SECONDS_IN_A_DAY;
 const SECONDS_IN_A_MONTH = 30 * SECONDS_IN_A_DAY;
 
-const defaultValues = {
-  royalty: 10,
-  editionSize: 100,
-  duration: SECONDS_IN_A_WEEK,
-  radius: 1, // In kilometers
-  hasAcceptedTerms: false,
-  notSafeForWork: false,
-};
-
 const durationOptions = [
   { label: "1 day", value: SECONDS_IN_A_DAY },
   { label: "1 week", value: SECONDS_IN_A_WEEK },
   { label: "1 month", value: SECONDS_IN_A_MONTH },
 ];
 
-const dropValidationSchema = yup.object({
-  file: yup.mixed().required("Media is required"),
-  title: yup.string().required().max(255),
-  description: yup.string().max(280).required(),
-  editionSize: yup
-    .number()
-    .required()
-    .typeError("Please enter a valid number")
-    .min(1)
-    .max(100000)
-    .default(defaultValues.editionSize),
-  royalty: yup
-    .number()
-    .required()
-    .typeError("Please enter a valid number")
-    .max(69)
-    .default(defaultValues.royalty),
-  hasAcceptedTerms: yup
-    .boolean()
-    .default(defaultValues.hasAcceptedTerms)
-    .required()
-    .isTrue("You must accept the terms and conditions."),
-  notSafeForWork: yup.boolean().default(defaultValues.notSafeForWork),
-  googleMapsUrl: yup.string().url(),
-  radius: yup.number().min(0.01).max(10),
-});
-
 // const { useParam } = createParam<{ transactionId: string }>()
 const DROP_FORM_DATA_KEY = "drop_form_local_data_free";
-
+const defaultValues = {
+  royalty: 10,
+  editionSize: 100,
+  duration: SECONDS_IN_A_WEEK,
+  radius: 1, // In kilometers
+  hasAcceptedTerms: false,
+  notSafeForWork: false,
+};
 export const DropFree = () => {
   const isDark = useIsDarkMode();
   const { rudder } = useRudder();
+  const { data: userProfile } = useMyInfo();
+
+  const isVerified = false;
+  userProfile?.data?.profile.has_verified_phone_number &&
+    userProfile?.data?.profile.verified;
```

Comment:
This line is wrong. isVerified is always false, the condition is written wrong

Reply:
Should be:

```tsx
  const isVerified =  userProfile?.data?.profile.has_verified_phone_number && userProfile?.data?.profile.verified;
```

Reply:
oh damn i just do a test, will remove it

Reply:
yeah, it's should be 
`const isVerified =  userProfile?.data?.profile.has_verified_phone_number || userProfile?.data?.profile.verified;`  
because they all belong to verified.